src/main/kotlin/org/pkl/lsp/services/PklProjectManager.kt

```diff
@@ -75,15 +75,15 @@ class PklProjectManager(project: Project) : Component(project) {
     if (file !is FsFile) null else pklProjects[file.projectKey]
 
   /** Tracks when PklProject files get added or deleted from the workspace. */
-  val addedOrRemovedFilesModificationTracker = SimpleModificationTracker()
+  val addedOrRemovedFilesModificationTracker =
+    project.pklFileTracker.filter { event -> event.files.any { it.path.endsWith("PklProject") } }
```

Comment:
This is not related to glob support, right?

Reply:
This is only related in that I needed to introduce another file tracker that just tracked when Pkl files get added or removed from the workspace. We already have other trackers for added/removed files, so I introduced a tracker just for adding/removing Pkl files, and made this one simply derived from it.